src/main/resources/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategy/checkPasswordRequested.js

```diff
@@ -46,8 +46,13 @@ Behaviour.specify(".specific-user-authorization", "checkPasswordRequired", 0, fu
   var useridField = findFormItem(e, "userid", findChild(e));
   var passwordField = findFormItem(e, "password", findChild(e));
   var passwordFieldBlock = findAncestor(passwordField, "TR");
+  if (passwordFieldBlock == null) {

```

Comment:
Would it be possible to get a commen explaining why this is here and why it works?

Reply:
Sure. Done.